src/mongodb/scores.service.ts

```
@@ -2299,15 +2377,16 @@ export class ScoresService {
     return { constructText, reptitionCount }
   }
 
-  async getTextMetrics(original_text: string, response_text: string, language: string, base64_string) {
+  async getTextMetrics(original_text: string, response_text: string, language: string, base64_audio: string) {
```

ESLint

[error] Replace `original_text:·string,·response_text:·string,·language:·string,·base64_audio:·string` with `⏎····original_text:·string,⏎····response_text:·string,⏎····language:·string,⏎····base64_audio:·string,⏎··`

(prettier/prettier)